samples/writer-test/writer-test.py

```diff
@@ -159,7 +159,7 @@ def runstrategy():
 
     # Plot if requested
     if args.plot:
-        cerebro.plot(numfigs=args.numfigs, volume=False, zdown=False)
+        cerebro.plot(numfigs=int(args.numfigs), volume=False, zdown=False)
```

Comment:
It is probably cleaner to specify the correct type upon defining the command line argument, instead of fixing the wrong argument type: 

parser.add_argument('--numfigs', '-n', default=1, type=int, ...